src/examples/logrotate

```diff
@@ -8,5 +8,6 @@
     delaycompress
     postrotate
         test -r /var/run/sssd.pid && /bin/kill -HUP `cat /var/run/sssd.pid 2>/dev/null` &>/dev/null || true
+        pgrep -c sssd_kcm &>/dev/null && /bin/kill -HUP `pgrep sssd_kcm 2>/dev/null` &>/dev/null || true
```

Comment:
Hi,

wouldn't it be possible to use `pkill` directly instead of the combination of `pgrep` and `kill`?

bye,
Sumit

Reply:
JFTR pkill is also not present in our dependency tree.

T.

Reply:
> wouldn't it be possible to use `pkill` directly instead of the combination of `pgrep` and `kill`?

Yes, of course. It escaped from my mind.

> JFTR pkill is also not present in our dependency tree.

Neither is `pgrep`, so I guess I'd better add it. Thanks for the heads-up.

Reply:
Thanks.